com.github.chipmuenk.pyfda.yaml

```diff
@@ -0,0 +1,258 @@
+app-id: com.github.chipmuenk.pyfda
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+# when the runtime version changes, check if python version changed, because in pyqt5 the
+# python version is hardcoded!
+command: pyfdax
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--device=dri"
+- "--own-name=com.github.chipmuenk.pyfda"
+- "--filesystem=home"
+rename-icon: pyfda_icon #Image will renamed to match the app-id konvention
+rename-appdata-file: pyfda.appdata.xml
+rename-desktop-file: pyfda.desktop
```

Comment:
these should be above the finish-args (as a common practice)

Reply:
done